src/helpers/getByAPI.js

```diff
@@ -20,9 +27,11 @@ const getNodeByText = (node, text) => {
       const textChildren = getChildrenAsText(node.props.children, Text);
       if (textChildren) {
         const textToTest = textChildren.join('');
-        return typeof text === 'string'
-          ? text === textToTest
-          : text.test(textToTest);
+        const { exact, normalizer } = options;
+        const normalizerFn = makeNormalizer({
+          normalizer,
+        });
```

Comment:
Looks like const normalizeFn = `normalizer ?? getDefaultNormalizer()` would be more straightforward.